src/main/ngapp/src/app/auth/auth.service.ts

```diff
@@ -39,40 +39,15 @@ export class AuthService {
   }
 
   // Login function.
-  login(email: string, password: string): void {
-    this.afAuth
-      .signInWithEmailAndPassword(email, password)
-      .then((res: any) => {
-        localStorage.setItem('user', JSON.stringify(res.user));
-        this.router.navigate(['/chat']);
-      })
-      .catch((error: any) => {
-        console.log('Something is wrong:', error.message);
-        window.alert(error.message);
-      });
+  login(email: string, password: string): Promise<any> {
+    return this.afAuth.signInWithEmailAndPassword(email, password)
   }
 
   // Registration function.
-  register(user: any) {
-    this.afAuth
-    .createUserWithEmailAndPassword(user.email, user.password)
-    .then((res) => {
-        localStorage.setItem('user', JSON.stringify(res.user));
-        
-        const userInstance : LocalUser = {
-                userId : res.user.uid,
-                name : user.firstName + " " + user.lastName,
-                email: user.email,
-                language: user.language
-        };
-
-        const promise = this.http.post<LocalUser>("/user", userInstance).toPromise();
-        promise.then(response => {
-            this.router.navigate(['/chat']);
-        });
```

Comment:
where is the new http POST method?

Reply:
In the request.ts file. After we sign up, we make a call to addUser()

Reply:
nice. seen it